server/pom.xml

```diff
@@ -63,7 +63,7 @@
     </dependency>
     <dependency>
      <groupId>org.eclipse.hono</groupId>
-     <artifactId>hono-config</artifactId>
+     <artifactId>hono-demo-certs</artifactId>
```

Comment:
You are right ... I needed to create a permissions.json under server/src/test/resources in order to have tests working. I removed the dependency and all works still fine ;)